python/ThirteenTeV/Higgs/Hadronizer_TuneCP5_13TeV_powhegEmissionVeto_1p_HToGDarkG_M-125_LHE_pythia8_cff.py

```diff
@@ -0,0 +1,33 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.Pythia8PowhegEmissionVetoSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
+                         maxEventsToPrint = cms.untracked.int32(1),
+                         pythiaPylistVerbosity = cms.untracked.int32(1),
+                         filterEfficiency = cms.untracked.double(1.0),
+                         pythiaHepMCVerbosity = cms.untracked.bool(False),
+                         comEnergy = cms.double(13000.),
+                         PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        pythia8PowhegEmissionVetoSettingsBlock,
+        processParameters = cms.vstring(
+            'POWHEG:nFinal = 1',   ## Number of final state particles
+                                   ## (BEFORE THE DECAYS) in the LHE
+                                   ## other than emitted extra parton
+            '3000022:all = DarkPhoton void 3 0 0 0. 0. 0. 0. 0.',
+            '3000022:isVisible = 0',
+            '25:m0 = 125.0',
+            '25:oneChannel = 1 1 100 22 3000022'
+          ),
+        parameterSets = cms.vstring('pythia8CommonSettings',
+                                    'pythia8CUEP8M1Settings',
```

Comment:
need to update this

Reply:
Good catch, thank you. Fixed in 8feae8b.